.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Critical CSS
+
+on: push
```

Comment:
Do we want to run it on pull_request only?

Reply:
I think since it's a public repo(free) we might be fine with `push`. I was primarily using this to test a [test branch](https://github.com/iFixit/css-gather/compare/testing-penthouse-preferred-store) without creating a pull.